src/openrct2/EditorObjectSelectionSession.cpp

```diff
@@ -342,9 +342,13 @@ static void window_editor_object_selection_select_default_objects()
 {
     if (_numSelectedObjectsForType[0] == 0)
     {
-        for (const auto& defaultSelectedObject : DefaultSelectedObjects)
+        for (auto defaultSelectedObject : DefaultSelectedObjects)
```

Comment:
This should still be `const`

Reply:
This was marked as resolved but still isn't `const` as before